torch_xla/core/xla_env_vars.py

```diff
@@ -2,6 +2,7 @@
 TPU_NUM_DEVICES = 'TPU_NUM_DEVICES'
 GPU_NUM_DEVICES = 'GPU_NUM_DEVICES'
 CPU_NUM_DEVICES = 'CPU_NUM_DEVICES'
+NEURONCORE_NUM_DEVICES = 'NEURONCORE_NUM_DEVICES'
```

Comment:
this env var is not being used anywhere? If we really need it can it just be `NEURON` since the device type is `NEURON`.

Reply:
Removed. Thanks.